plugins/capacitor-app-migrations/skills/cordova-to-capacitor/SKILL.md

```
@@ -0,0 +1,546 @@
+cordova plugin list | xargs -I {} cordova plugin remove {}
+
+# Remove config.xml (after backing up)
+mv config.xml config.xml.backup
+```
+
+## Common Issues and Solutions
+
+### Issue: Plugin Not Found
+
+**Problem:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)